ui/packages/consul-ui/app/models/topology.js

```diff
@@ -19,23 +19,27 @@ export default class Topology extends Model {
   @attr() Downstreams; // Service[],
   @attr() meta; // {}
 
-  @computed('Upstreams', 'Downstreams')
+  @computed('Downstreams')
```

Comment:
When I did this computed function I forgot undefined intentions only exist for Downstream services.